ludwig/utils/upload_utils.py

```diff
@@ -256,17 +261,37 @@ def upload(
         )
 
         # Upload all artifacts in model weights folder
-        upload_path = self.api.upload_folder(
+        commit_message_weights: str | None = f"{commit_message} (weights)" if commit_message else commit_message
+        commit_description_weights: str | None = (
+            f"{commit_description} (weights)" if commit_description else commit_description
+        )
+        upload_path_weights = self.api.upload_folder(
```

Comment:
@sanjaydasgupta Is there a type hint we could add to `upload_path_weigths`?  Thank you.